sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -347,6 +347,15 @@ querySpecification
        windows?)
     ;
 
+hint
+    : '/*+' hintStatement '*/'
+    ;
+
+hintStatement
+    : hintName=identifier identifierList
```

Comment:
we could change this to: `hintName=identifier '(' parameters+=identifier (',' parameters+=identifier)* ')'`

Then we would only need to treat one case in the AstBuilder :)